examples/csharp/README.md

```diff
@@ -2,31 +2,39 @@
 
 ## Install the onnxruntime-genai library
 
-* Install the python package
-
-  ```bash
-  pip install onnxruntime-genai
-  ```
 
 ## Get the model
 
-Install the model builder script dependencies
+You can generate the model using the model builder this library, or bring your own model.
+
+If you bring your own model, you need to provide the configuration. See the [config reference](https://onnxruntime.ai/docs/genai/reference/config).
+
+To generate the model with model builder:
 
-```bash
-pip install numpy
-pip install transformers
-pip install torch
-pip install onnx
-pip install onnxruntime
-```
+1. Install the python package
 
-Run the model builder script to export, optimize, and quantize the model. More details can be found [here](../../src/python/py/models/README.md)
+   ```bash
+   pip install onnxruntime-genai
```

Comment:
we need to add the URL before we publish to pypi

Reply:
Fixed by directing to install instructions.